composer.json

```diff
@@ -11,7 +11,7 @@
         }
     ],
     "require": {
-        "php": ">=5.6.3",
+        "php": ">=5.5",
```

Comment:
I may be missing something but the code here seems to run on php 5.5 just fine as well as all the dependencies running on php 5.5. Running the latest and greatest is cool but no need to require it in this case